strictdoc/export/html/form_objects/grammar_form_object.py

```diff
@@ -28,22 +28,33 @@ def __init__(
         field_mid: str,
         field_name: str,
         document_mid: str,
+        field_human_title: Optional[str] = None,
     ):
         self.field_mid: str = field_mid
         self.field_name: str = field_name
         self.document_mid: str = document_mid
+        self.field_human_title: Optional[str] = field_human_title
 
     @staticmethod
-    def create(*, grammar_element: GrammarElement, document_mid: str):
+    def create(
+        *,
+        grammar_element: GrammarElement,
+        document_mid: str,
+        field_human_title: Optional[str] = None,
+    ):
         return GrammarElementFormField(
             field_mid=grammar_element.mid,
             field_name=grammar_element.tag,
             document_mid=document_mid,
+            field_human_title=field_human_title,
```

Comment:
Since this is a case when a new grammar element form field is created we are good to have it explicitly to None:

```
field_human_title=None
```

(StrictDoc's code style is rather verbose with the added value of a maximum transparency. The default parameters are almost never used.)